tscached/kquery.py

```diff
@@ -22,6 +24,16 @@ def from_request(cls, request, redis_client):
         for metric in request.get('metrics', []):
             new = cls(redis_client)
             new.query = metric
+
+            # This seemingly terrifying kludge makes it easy to merge chunks of MTS together.
+            # For all aggregators, we remove align_sampling and add align_start_time.
+            # Otherwise, partial windows (if aggregated) will appear on every seam in the data. Info:
+            # https://kairosdb.github.io/docs/build/html/restapi/QueryMetrics.html#metric-properties
+            for agg_ndx in xrange(len(new.query.get('aggregators', []))):
```

Comment:
To the first point, yeah. This is sum, diff, sample, etc.

To the second, oh hell yeah. It's probably scope creep for now. My initial intuition was to have tscached be as naive as possible to what the user wants. Just take their word for it as best we can?